contracts/Staking.sol

```diff
@@ -201,6 +212,20 @@ contract Staking is Pauser, Whitelist {
         emit ValidatorStatusUpdate(_valAddr, dt.ValidatorStatus.Unbonded);
     }
 
+    function rewardValidator(address _valAddr, uint256 _tokens) public {
```

Comment:
My primary concern is that any require() fails (after ligthClient check0 may result in the issue that the LightClient can never process new epoch headers.